content/doc/book/pipeline/syntax.adoc

```diff
@@ -306,16 +305,17 @@ pipeline {
         }
     }
 }
-// Script //
 ----
 <1> Execute all the steps defined in this Pipeline within a newly created container
 of the given name and tag (`maven:3-alpine`).
+=====
 
 ====== Stage-level `agent` section
 
-[pipeline]
+.Stage-level Agent Section
+=====
```

Comment:
It looks like these headings are doubled, and the second one just appears as plain text. I'm not sure if that is intended, just a problem in the GitHub preview, or a real problem:

<img width="311" alt="Screen Shot 2019-12-16 at 09 46 27" src="https://user-images.githubusercontent.com/1068968/70916029-f3b7bc80-1fe8-11ea-8bb5-1a7bddc84632.png">

Reply:
@dwnusbaum thanks, duplicate removed. The plaintext rendering only happens at GitHub. A little bit better preview is at https://zbynek.github.io/jenkins.io/order-syntax/doc/book/pipeline/syntax/